src/components/Clients/Brandings.tsx

```diff
@@ -3,6 +3,7 @@ import { useEffect, useState } from 'react'
 import naturalEcoFriendly from './../../assets/images/naturaleza-ecoamigable.svg'
 import oasisRestaurant from './../../assets/images/logo-oasis-op.webp'
 import { BrandingsWrapper } from './Clients.styles'
+import { LazyLoadImage } from 'react-lazy-load-image-component'
```

Comment:
You need white space between local and external imports

Reply:
Done!